wordpress-mobile/WordPress-iOS
# Zendesk: Update to 5.1.1

Fixes #n/a

This updates the `ZendeskSupportSDK` to version 5.1.1. 

To test:

I perused the [release notes](https://developer.zendesk.com/embeddables/docs/ios-unified-sdk/release_notes) between 5.0.0 and 5.1.1, and tested functionality. I saw nothing worth noting. So just some smoke testing should do it.
- Go to Me > Help & Support.
- If you've set a `Contact Email`, verify it's the one you want to use.
- Select `Contact Support`.
- If prompted (if you've not set a `Contact Email`), enter your contact info.
- Submit a request.
- Go to Zendesk, and verify a ticket was created.
- In the app, go to `My Tickets`.
- Verify your newly created issue appears there.

Note: This is listed as a ZD [Known Issue](https://developer.zendesk.com/embeddables/docs/ios-unified-sdk/known_issues): `Missing dSYMs for iOS simulator architecture causes a warning to be emitted for CocoaPods integrators. Note: This warning is emitted only when building apps on iOS Simulators.` I didn't see any warnings, but if you do that's why.

PR submission checklist:
